RepoDb.Core/RepoDb/IIdOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace RepoDb
+{
+    public interface IIdOptions<T> where T : class
```

Comment:
Can we rename this to `IIdentityOptions`?

Reply:
Oh sorry, It should have been `IIdentityOptions`. I forgot to change it